ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -1508,6 +1519,40 @@ abstract class AbstractHttpServiceIntegrationTest
       }: Future[Assertion]
   }
 
+  "fetch fails when readAs not authed, even if prior fetch succeeded" in withHttpService {
```

Comment:
maybe also worth adding a test that it returns the expected result if its correctly authorized?

Reply:
There's no interestingly different codepath that is followed when that is the case; I doubt the value of such a test.